posts/2021/07/numpy-accessibility-guidelines.md

```diff
@@ -0,0 +1,180 @@
+<!--
+.. title: Making Numpy Accessible: Guidelines and Tools .. slug:
```

Comment:
If you have a look at how this metadata is done in other `.md` files, that should fix the preview build error.

Looking forward to reading this post!

Reply:
Thank you for catching, it looks like the metadata got squashed when I applied text-wrapping. It's fixed and passes the latest deployment check.